### ISSUE TITLE
Typo disallows export for Time::Moment::Adjusters

### ISSUE THREAD

**Issue by unobe** [chansen/p5-time-moment#29]
Thanks for the module, @chansen ! I use it at work and am also considering using it for a custom project and noted that there's a misspelling of 'OrthodoxEasterSunday' on [line 17 of Time::Moment::Adjusters](https://metacpan.org/source/CHANSEN/Time-Moment-0.43/lib/Time/Moment/Adjusters.pm#L17) that doesn't allow the function to be exported.

**Comment by chansen** [comment]
Thank you for the bug report! I'll push a CPAN release this weekend, if you need a temporary workaround you can use the following:

```perl
use Time::Moment;
use Time::Moment::Adjusters qw[:all];

BEGIN {
  # Workaround typo in Time::Moment <= v0.43
  if (__PACKAGE__->can('OthodoxEasterSunday')) {
    *OrthodoxEasterSunday = \&Time::Moment::Adjusters::OrthodoxEasterSunday;
  }
}

say Time::Moment->now->with(OrthodoxEasterSunday);
```

**Comment by unobe** [comment]
Thanks @chansen !

**Comment by chansen** [comment]
I have shipped the long overdue release of [v0.44 to CPAN](https://metacpan.org/release/CHANSEN/Time-Moment-0.44) incorporating fixes for this typo. Sorry for the delay.

--
chansen

**Comment by unobe** [comment]
Thanks @chansen !
